nubison_model/Model.py

```diff
@@ -138,6 +138,8 @@ def register(
     model_name: Optional[str] = None,
     mlflow_uri: Optional[str] = None,
     artifact_dirs: Optional[str] = None,
+    params: Optional[dict[str, Any]] = None,
+    metrics: Optional[dict[str, float]] = None,
```

Comment:
How can we handle cases where metrics need to be in the form of a list or a numpy.ndarray, such as metrics = {"loss_curve": [0.5, 0.3, 0.2, 0.1]}?

Reply:
I think you think the metric as an array for time-based values. But I intent it serves to record final metrics. The goal of this project is to facilitate easy model storage rather than to focus on experiment tracking. If we expand this project to include training, I believe the metrics could be as an array.